bin/db_setup.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
 for db in osu osu_store osu_mp osu_chat osu_charts osu_updates; do
-  echo "CREATE DATABASE IF NOT EXISTS ${db} DEFAULT CHARSET utf8mb4" | mysql -u root
-  echo "CREATE DATABASE IF NOT EXISTS ${db}_test DEFAULT CHARSET utf8mb4" | mysql -u root
+  echo "CREATE DATABASE IF NOT EXISTS ${db} DEFAULT CHARSET utf8mb4" | mysql -u root $@
+  echo "CREATE DATABASE IF NOT EXISTS ${db}_test DEFAULT CHARSET utf8mb4" | mysql -u root $@
```

Comment:
doesn't really affect how it's currently used but it should be quoted for correct usage
```suggestion
  echo "CREATE DATABASE IF NOT EXISTS ${db} DEFAULT CHARSET utf8mb4" | mysql -u root "$@"
  echo "CREATE DATABASE IF NOT EXISTS ${db}_test DEFAULT CHARSET utf8mb4" | mysql -u root "$@"
```